generator/integration_tests/golden/database_admin_connection.gcpcxx.pb.cc

```diff
@@ -434,7 +440,8 @@ class DatabaseAdminConnectionImpl : public DatabaseAdminConnection {
         backoff_policy_prototype_->clone());
     auto idempotency = idempotency_policy_->ListBackups(request);
     char const* function_name = __func__;
-    return google::cloud::internal::MakePaginationRange<ListBackupsRange>(
+    return google::cloud::internal::MakePaginationRange<StreamRange<
```

Comment:
Can we change the line break here to: 

```cc
    return google::cloud::internal::MakePaginationRange<
        StreamRange<::google::test::admin::database::v1::Backup>>(
```

Which I think is what clang-format would give us.